tools/migration/ppaas-artifact-converter/src/main/bin/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo "-------------------------------------------------------"
+echo "STARTING PPaaS ARTIFACT MIGRATION TOOL (4.0.0 to 4.0.1)"
```

Comment:
I think it would be better to use standard convention here rather than using uppercase letters.